sw/device/silicon_creator/lib/boot_svc/boot_svc_next_boot_bl0_slot.h

```diff
@@ -0,0 +1,105 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_NEXT_BOOT_BL0_SLOT_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_NEXT_BOOT_BL0_SLOT_H_
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/silicon_creator/lib/base/chip.h"
+#include "sw/device/silicon_creator/lib/boot_svc/boot_svc_header.h"
+#include "sw/device/silicon_creator/lib/error.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+enum {
+  /*
+   * Encoding generated with
+   * $ ./util/design/sparse-fsm-encode.py -d 6 -m 2 -n 32 \
+   *     -s 2121036560 --language=c
+   *
+   * Minimum Hamming distance: 14
+   * Maximum Hamming distance: 14
+   * Minimum Hamming weight: 11
+   * Maximum Hamming weight: 13
+   */
+  kBootSvcNextBootBl0SlotA = 0x08c0d499,
+  kBootSvcNextBootBl0SlotB = 0x7821e03a,
```

Comment:
Is there a better way to represent the slot we'd like besides these enums?

Reply:
Not really, these constants are good. I also checked but I don't think we define a constant for these yet. Thanks @milesdai !